pinot-core/src/main/java/org/apache/pinot/core/operator/query/FastFilteredCountOperator.java

```diff
@@ -38,14 +38,16 @@ public class FastFilteredCountOperator extends BaseOperator<IntermediateResultsB
   private final BaseFilterOperator _filterOperator;
   private final AggregationFunction[] _aggregationFunctions;
   private final SegmentMetadata _segmentMetadata;
+  private final boolean _nullHandlingEnabled;
 
   private long _docsCounted;
 
   public FastFilteredCountOperator(AggregationFunction[] aggregationFunctions, BaseFilterOperator filterOperator,
-      SegmentMetadata segmentMetadata) {
+      SegmentMetadata segmentMetadata, boolean nullHandlingEnabled) {
```

Comment:
This operator cannot support null handling. We should not use it when null handling is enabled